infra/README.md

```diff
@@ -0,0 +1,58 @@
+
+# OpenSearch cluster to test the interceptor at development time
```

Comment:
I would just say "Integration Tests" and "This project creates an OpenSearch cluster on AWS for development purposes ..."